2.5/Dockerfile.rhel7

```diff
@@ -0,0 +1,63 @@
+FROM rhscl/s2i-base-rhel7:1
+
+# This image provides a Ruby environment you can use to run your Ruby
+# applications.
+
+EXPOSE 8080
+
+ENV RUBY_MAJOR_VERSION 2
+ENV RUBY_MINOR_VERSION 5
+ENV RUBY_VERSION "${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}"
+ENV RUBY_SCL_NAME_VERSION "${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}"
+ENV RUBY_SCL "rh-ruby${RUBY_SCL_NAME_VERSION}"
+ENV RUBY_NODEJS_SCL "rh-nodejs6"
+ENV IMAGE_NAME "rhscl/ruby-${RUBY_SCL_NAME_VERSION}-rhel7"
```

Comment:
Please join these into a single ENV command (the one that holds `SUMMARY` and `DESCRIPTION` now). Every Dockerfile command you use (unnecessarily) introduces a new layer to the resulting image and it is a good idea to keep those minimal.

Reply:
You meant like this?
This declaration style is problem in this case.
Because the timing of the evaluation of variables is only once in below style.
For example `${RUBY_MAJOR_VERSION}` is not "2" but empty string in below case.

```
ENV RUBY_MAJOR_VERSION=2
    RUBY_MINOR_VERSION=5
    RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}"
    RUBY_SCL_NAME_VERSION="${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}"
    RUBY_SCL="rh-ruby${RUBY_SCL_NAME_VERSION}"
    RUBY_NODEJS_SCL="rh-nodejs6"
    IMAGE_NAME="rhscl/ruby-${RUBY_SCL_NAME_VERSION}-rhel7"
```

Reply:
Ah right, I have missed you are re-using the variables right away. In that case you can at least minimize the layers by creating groups of ENV declarations:

```
ENV RUBY_MAJOR_VERSION=2 \
    RUBY_MINOR_VERSION=5 \
    RUBY_NODEJS_SCL="rh-nodejs6"

ENV  RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}"  \
    RUBY_SCL="rh-ruby${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}"  \
    IMAGE_NAME="rhscl/ruby-${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}-rhel7"
```

Reply:
ok, I will modify like you suggested.